packages/app/src/components/Root/Root.tsx

```diff
@@ -88,7 +89,9 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
     <Sidebar>
       <SidebarLogo />
       <SidebarGroup label="Search" icon={<SearchIcon />} to="/search">
-        <SidebarSearchModal />
+        <SidebarSearchModal>
```

Comment:
Getting this in the example app is nice!  I wonder if it'd be good to document elsewhere too.  Perhaps storybook?  The sidebar story could use a little love, perhaps.